test/dicer-multipart-extra-trailer.spec.js

```diff
@@ -55,6 +55,8 @@ describe('dicer-multipart-extra-trailer', () => {
         })
       }).on('error', function (err) {
         error = err
+      }).on('trailer', function (data) {
+        assert(data.toString() === 'Extra', 'trailer should contain the extra data')
```

Comment:
we probably should count assertions somewhere, otherwise we may simply be skipping this event altogether, and test would still pass.

Reply:
added